.github/workflows/pre-commit-check.yml

```diff
@@ -1,6 +1,9 @@
 name: Pre Commit Check
 on:
   pull_request:
+    types:
+      - opened
+      - synchronize
```

Comment:
Do we need this?

Reply:
I should have added a comment: with this, the "Pre Commit Check" will *not* be run when we label/unlabel the PR, which I trust is what we want